examples/chart/teleport-daemonset/templates/clusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
I'm not sure if I understand the need for a `ClusterRole`. This is just running a privileged container as a `DaemonSet`, why can't this be namespaced?